examples/weather/unified_recipe/train.py

```diff
@@ -51,9 +51,10 @@
 from modulus.utils import StaticCaptureEvaluateNoGrad, StaticCaptureTraining
 
 from seq_zarr_datapipe import SeqZarrDatapipe
+from model_packages import save_inference_model_package
 
 
-def batch_normalized_mse(pred: Tensor, target: Tensor) -> Union[Tensor, float]:
```

Comment:
1. My understanding is that `batch_normalized_mse` expects `Tensor`s as input, so why remove the type hint?
2. The function returns the result of `torch.mean`, which is actually a `Tensor`, not `float`, so the type hint is incorrect. If you absolutely must return `float` you should use `.item()` but note that it will cause a sync.

Reply:
Fixed this